src/main/java/com/um/inventoryservice/BusinessLayer/OrderServiceImpl.java

```diff
@@ -38,4 +38,13 @@ public Flux<OrderDTO> getOrdersByClientId(String clientId){
         return orderRepository.findOrdersByClientId(clientId)
                 .map(EntityDTOUtil::toDTO);
     }
+
+    @Override
+    public Flux<StockOrderDTO> getStockOrdersByQuantity() {
+        return orderRepository.findAll()
+                .flatMapIterable(order -> order.getStockOrderDTOS())
```

Comment:
What do these 2 lines do? The flatMapIterable and the sorting.

Reply:
The flatmapIterable flattens the stockOrderDTO list in each order to a stream then iterates through each one, the sort rearranges the stockOrderDTOs in descending order of quantity, or by most sold.